docs/release_notes/v1.7.0.adoc

```diff
@@ -0,0 +1,68 @@
+[id="{p}-release-notes-v1.7.0"]
+=== Elastic Cloud Control (ecctl) v1.7.0
+++++
+<titleabbrev>v1.7.0</titleabbrev>
+++++
+
+<<{p}-release-notes-v1.7.0-whats-new,What's new>> | <<{p}-release-notes-v1.7.0-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.7.0-breaking-changes,Breaking changes>> | <<{p}-release-notes-v1.7.0-changelog,Changelog>>
+
+Welcome to the v1.7.0 release of {n}. This version brings new features, some breaking changes, and bug fixes.
+
+Elastic Cloud Control (ecctl) is Elastic’s CLI to manage Elasticsearch Service (ESS) and Elastic Cloud Enterprise (ECE).
+
+Download the release binaries:
+
+[%hardbreaks]
+https://download.elastic.co/downloads/ecctl/1.7.0/ecctl_1.7.0_darwin_amd64.tar.gz[ecctl_1.7.0_darwin_amd64.tar.gz]
+https://download.elastic.co/downloads/ecctl/1.7.0/ecctl_1.7.0_linux_32-bit.deb[ecctl_1.7.0_linux_32-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.7.0/ecctl_1.7.0_linux_32-bit.rpm[ecctl_1.7.0_linux_32-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.7.0/ecctl_1.7.0_linux_386.tar.gz[ecctl_1.7.0_linux_386.tar.gz]
+https://download.elastic.co/downloads/ecctl/1.7.0/ecctl_1.7.0_linux_64-bit.deb[ecctl_1.7.0_linux_64-bit.deb]
+https://download.elastic.co/downloads/ecctl/1.7.0/ecctl_1.7.0_linux_64-bit.rpm[ecctl_1.7.0_linux_64-bit.rpm]
+https://download.elastic.co/downloads/ecctl/1.7.0/ecctl_1.7.0_linux_amd64.tar.gz[ecctl_1.7.0_linux_amd64.tar.gz]
+
+[float]
+[id="{p}-release-notes-v1.7.0-whats-new"]
+==== What's new
+
+// TODO: FILL OR REMOVE
+
+[float]
+[id="{p}-release-notes-v1.7.0-bug-fixes"]
+==== Bug fixes
+
+// TODO: FILL OR REMOVE
+
+[float]
+[id="{p}-release-notes-v1.7.0-breaking-changes"]
+==== Breaking changes
+
+// TODO: FILL OR REMOVE
```

Comment:
Shall we keep these sections even if they are empty?

Reply:
@marclop @elastic/cloud-delivery what should we do?

Reply:
should remove anything that's empty.